assets/src/editorIndex.js

```diff
@@ -66,3 +66,6 @@ registerBlockVariation('core/group', {
   },
   icon: 'admin-links',
 });
+
+import { registerBlockTemplates } from './block-templates/register';
```

Comment:
Not a big deal, but maybe this `import` statement should be with all the others at the beginning of the file?